source/text/WrappedText.cpp

```diff
@@ -245,6 +255,8 @@ void WrappedText::Wrap()
 
 				// Adjust the spacing of words in the now-complete line.
 				AdjustLine(lineBegin, lineWidth, false);
+				// This is a full width line, so mark it as such.
+				longestWidth = lineWidth;
```

Comment:
Shouldn't this be done like the others before the `AdjustLine` call? Because `lineWidth` is unconditionally reset to 0 after the call.

Reply:
Good catch. I moved the longestWidth computation *into* `AdjustLine` to fix this, and also make it so that I can take alignment into account.